packages/teleport-plugin-vue-css/src/index.ts

```diff
@@ -44,7 +44,7 @@ export const createPlugin: ComponentPluginFactory<VueStyleChunkConfig> = (config
     traverseElements(node, (element) => {
       const { style, key } = element
 
-      if (style) {
+      if (style && Object.keys(style).length > 0) {
```

Comment:
the same comment here as for css-modules. when splitting dynamic and static styles, we should check the separate objects so that we only generate what is actually needed